src/OrbitGl/GraphTrack.h

```diff
@@ -45,6 +45,11 @@ class GraphTrack : public Track {
   virtual void AddValues(uint64_t timestamp_ns, const std::array<double, Dimension>& values) {
     series_.AddValues(timestamp_ns, values);
   }
+
+  uint32_t GetNumTimers() const override;
```

Comment:
`[[nodiscard]]` for this and the following `Get*` methods

Reply:
done.